frontend/packages/ceph-storage-plugin/integration-tests-cypress/tests/multiple-pool.spec.ts

```diff
@@ -1,9 +1,61 @@
+import { modal } from '../../../integration-tests-cypress/views/modal';
 import { checkErrors } from '../../../integration-tests-cypress/support';
-import { storagePool, PoolState } from '../views/multiple-pool';
+import { POOL_PROGRESS } from '../../src/constants/storage-pool-const';
+import {
+  poolName,
+  replicaCount,
+  populateBlockPoolform,
+  deleteBlockPoolFromCli,
+} from '../views/block-pool';
 
 // Pool var
-const poolName: string = 'example.pool';
-const replicaCount: string = '2';
+const storagePoolDropdown: string = 'pool-dropdown-toggle';
+
+const poolMessage: {
+  [key in POOL_PROGRESS]?: () => string;
+} = {
+  [POOL_PROGRESS.FAILED]: () => `Pool "${poolName}" already exists`,
+  [POOL_PROGRESS.CREATED]: () => `Pool ${poolName} was successfully created`,
+};
+
+const storagePool = {
+  prepareStorageClassForm: () => {
+    // Select provisioner
```

Comment:
Moved these part from multiple-pool.ts to here, and i delete views/multiple-pool.ts (since it is specific code to pool modal, i moved here)